frontend/components/MovingBus.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div class="absolute inset-0 -z-10 flex items-end">
+    <SvgJoinRoomBgTreesHouses class="w-screen" :fontControlled="false" filled />
+    <SvgJoinRoomBgBus
+      id="bus"
+      class="absolute bottom-[6.7%] w-[282px]"
+      :fontControlled="false"
+      filled
+    />
+  </div>
+</template>
+<script setup>
+const props = defineProps({
+  busStartingPosition: Number,
+  busEndingPosition: Number,
+  busSpeed: {
+    type: Number,
+    default: 2,
+  },
+});
+
+let id = null;
+
+function moveBus() {
+  const elem = document.getElementById("bus");
```

Comment:
abgesehen davon dass bitte css animation, document.getElementById tun wir nicht in Vue, da verwenden wir element refs